config/app.php

```diff
@@ -159,7 +159,7 @@
         Vatsim\Xml\XmlServiceProvider::class,
         Barryvdh\Debugbar\ServiceProvider::class,
         Barryvdh\LaravelIdeHelper\IdeHelperServiceProvider::class,
-        Maknz\Slack\SlackServiceProvider::class,
+//        Maknz\Slack\SlackServiceProvider::class,
```

Comment:
Can you expand on the error that this was throwing that led to removing it?

Reply:
The slack service provider utilises application::share which was removed two (?) Laravel versions ago. Unsure if it's even still being used?

Reply:
Doesn't appear to be used, can just remove the line